### PR TITLE
bpo-22062: Updated docstring and documentation for pathlib

### PR DESCRIPTION
**Pull request by eivl**
Original patch by Mike Short

https://bugs.python.org/issue22062

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


When your account is ready, please add a comment in this pull request
and a Python core developer will remove the `CLA not signed` label
to make the bot check again.

You can [check yourself](https://devguide.python.org/committing.html#checking-if-the-cla-has-been-received)
to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

File: Doc/library/pathlib.rst [hunk 1]
```diff
@@ -965,8 +965,8 @@ call fails (for example because the path doesn't exist):
 
 .. method:: Path.rglob(pattern)
 
-   This is like calling :meth:`Path.glob` with "``**``" added in front of the
-   given *pattern*::
+   This is like calling :func:`Path.glob` with "``**/``" added in front of the
+   given relative *pattern*:
```
Comment (encukou) [inline Doc/library/pathlib.rst]: The double colon (`::`) at the end is intentional; it's [ReST syntax for a code block](https://docs.python.org/3.0/documenting/rest.html#source-code). Please leave it there.

File: Misc/NEWS.d/next/Documentation/2018-07-28-12-41-01.bpo-22062.TaN2hn.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+documentation and docstrings updated, Original patch by Mike Short
```
Comment (encukou) [inline Misc/NEWS.d/next/Documentation/2018-07-28-12-41-01.bpo-22062.TaN2hn.rst]: Nitpick: The blurbs usually use complete sentences – start with a capital letter and put period (not comma) before the author notice. (You can see the [changelog](https://docs.python.org/3/whatsnew/changelog.html#changelog) for examples of how these are formatted.)

**Comment by csabella** [comment]
@eivl Would you be able to apply the changes requested by @encukou?  Thanks!

**Comment by eivl** [comment]
ofc, this has been totaly forgotten.... I will do it within tomorrow..

Den søn. 13. jan. 2019 kl. 02:33 skrev Cheryl Sabella <
notifications@github.com>:

**Review by encukou — APPROVED** [review]

**Pull request merged**